docs/content/preview/contribute/core-database/build-from-src-centos.md

```diff
@@ -87,9 +87,9 @@ There is no need to add any of those directories to `PATH`.
 
 ### Build release package
 
-You can build a release package by executing:
+Run the `yb_release` script to build a release package:
 
-```shell
+```output.sh
```

Comment:
What does this format specifier mean?

Reply:
it means "format as shell commands, but don't add a copy button"

Reply:
(`nocopy.<language>` does the same thing)